Makefile

```diff
@@ -429,7 +429,10 @@ $(eval $(call create_help_target,test-help,TEST_HELP_TEXT))
 test: test-python
 
 .PHONY: test-chapel
-test-chapel: $(TEST_TARGETS)
+.SILENT:
+test-chapel:
+	@echo $(shell python3 $(MODULE_GENERATION_SCRIPT) $(ARKOUDA_CONFIG_FILE));
```

Comment:
So far as I know the chapel tests don't need the config file, why is this generation needed?

Reply:
Looks like it was added in https://github.com/Bears-R-Us/arkouda/commit/e622dc5fbdc3c309ac8701f5f9929a95b53c4d47 -- @bmcdonald3 can you take an action item to look into whether this is needed.